Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,40 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+//Map the tasks to durations in hours.
+let Maartje = tasks.map(element => element.duration / 60);
+console.log(Maartje);
+//(9) [3, 2, 0.3333333333333333, 3.3333333333333335, 4, 3,
+// 0.16666666666666666, 3.3333333333333335, 0.6666666666666666]
+
+
+//Filter out everything that took less than two hours 
+let Maartje1 = Maartje.filter(element => element > 2);
```

Comment:
You should avoid naming variables like this at all costs! Always use names which actually represent what is going to be stored in the variable.